application-templates/starter/webpack.config.prod.js

```diff
@@ -1,5 +1,7 @@
 const path = require('path');
-const createWebpackConfigForProduction = require('@commercetools-frontend/mc-scripts/config/create-webpack-config-for-production');
+const {
+  createWebpackConfigForProduction,
+} = require('@commercetools-frontend/mc-scripts');
```

Comment:
Is the change in export here a breaking change or do we keep the default export for now?